gobbler-server.rkt

```diff
@@ -345,6 +347,7 @@ waiting list.
   ;; =====================================
 
   (check-equal? (queue-world WAITING0 iworld1) WAITING1)
+  (check-equal? (queue-world WAITING1 iworld2) WAITING2)
```

Comment:
We kind of already talked about this but you'll want to do the whole `let` situation in order to test these since its mutation

Reply:
That actually won't work; `let` doesn't clone the value like you said, it aliases it. For instance:

```racket
(struct posn [x y] #:mutable)
(define p (posn 3 4))

(let ([local-p p])
  (set-posn-x! p 5)
  (posn-x p))

(posn-x p)
```

Both calls to `posn-x` will evaluate to `5`. I documented my plan for a test fixture [here](https://github.com/tkindy/gobbler-server/issues/4#issuecomment-342981655) and I'm almost done implementing it.